Test/civl/inductive-sequentialization/BroadcastConsensus.bpl

```diff
@@ -47,13 +47,12 @@ function max(CH:[val]int) : val;
 function card(CH:[val]int) : int;
 
 axiom card(MultisetEmpty) == 0;
-axiom (forall v:val :: card(MultisetSingleton(v)) == 1);
 axiom (forall CH:[val]int, v:val :: card(MultisetPlus(CH, MultisetSingleton(v))) == card(CH) + 1);
```

Comment:
On my machine this axiom can be removed too.

Reply:
Done.